doc/en/mark.rst

```diff
@@ -42,8 +43,13 @@ Marks can be registered in ``pytest.ini`` like this:
         slow
         serial
 
-This can be used to prevent users mistyping mark names by accident. Test suites that want to enforce this
-should add ``--strict`` to ``addopts``:
+When the ``--strict`` command-line flag is passed, any unregistered marks
+applied with the ``@pytest.mark.name_of_the_mark`` decorator will trigger an
+error, including built-in marks such as ``skip`` or ``xfail``.
+
+Marks added by pytest or by a plugin instead of the decorator will not trigger
+the warning or this error.  Test suites that want to enforce a limited set of
+markers can add ``--strict`` to ``addopts``:
```

Comment:
Can we introduce `--strict-marks` as an alias to `--strict`?

Historically `--strict` started with marks but it was intended to also turn warnings into errors, but those plans have changed now that we use standard warnings, so I think it makes sense to make that explicit in the option name. Thoughts?

Reply:
i like to see --strict go away  as the current form is simply very lacking

Reply:
I'd prefer to leave that to a future PR, since I don't have time to do that too at the moment.

Reply:
Fair enough! 👍